_posts/2016-08-01-5-nusmods-features-you-might-not-know-about.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "5 NUSMods Features You Might Not Know About"
+date:   2016-07-31 14:59:26
+author: Tay Yang Shun
+author_thumbnail: 558978353
+---
+
+NUSMods has been an incredibly useful platform for NUS students, providing a user-friendly interface for planning their timetables and searching through the available modules. However, it does more than just that! Here are some of the lesser known features in NUSMods that can improve your productivity and make you even more productive!
+
+## Lesson Theme Colors and Dark Mode
+
+One of the most popular features of NUSMods is the colorful timetable lesson slots, as it gives colour to one's school life (literally). Do you know that there are a grand total of 32 different color themes for these lesson slots? They can be accessed via the [**Preferences**](https://nusmods.com/preferences) page or simply pressing the <kbd>←</kbd> and <kbd>→</kbd> arrow keys to switch between themes. Our personal favourites are the *Mocha*, *Monokai* and *Ocean* themes. There is even an [NUSWhispers](https://nuswhispers.com) theme if you can't get enough of it (;
```

Comment:
"our personal favourite" -> "my personal favourite", since this post is authored by you, it sounds more personal :)